### PR TITLE
Feat: Added products page to my e-com site.

### PR DESCRIPTION
**Pull request by zoef-yt**
Added products page to my e-com site.

### CODE REVIEW

File: screens/product-listing-page/product-listing-page.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+				<img
+					class="avatar avatar-sm"
+					src="https://media-exp1.licdn.com/dms/image/C4E03AQEm0ZBjaIr1hg/profile-displayphoto-shrink_200_200/0/1610261504692?e=1649894400&v=beta&t=kKO77EwqflEDsrZf5eG7xC7ZUB4hD_BkZuRJyFiFHbI"
+				/>
+			</div>
+		</nav>
+		<aside class="product-aside"> </aside>
+		<main class="product-main">
+			<div class="card">
+				<div class="card-overlay-holder">
```
Comment (itsmebipul00) [inline screens/product-listing-page/product-listing-page.html:75]: try not using pngs, jpgs and webps are lot lighter files

File: screens/product-listing-page/product-listing-page.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(20rem, 29rem));
+	grid-template-rows: minmax(min-content, max-content);
+	justify-content: space-evenly;
+	gap: 4rem;
+}
+/* 
+Commented for future use
+ 
+.card-img {
```
Comment (varun13169) [inline screens/product-listing-page/product-listing-page.css:33]: Get rid of redundant comments.

**Review by varun13169 — CHANGES_REQUESTED** [review]

**Review by itsmebipul00 — APPROVED** [review]

**Review by j24m — APPROVED** [review]
Both code and UI looks dope. 🔥 GTM!!

File: screens/product-listing-page/product-listing-page.css (hunk 2 above)
Comment (zoef-yt) [reply screens/product-listing-page/product-listing-page.css:33]: It is for the future.

File: screens/product-listing-page/product-listing-page.html (hunk 1 above)
Comment (zoef-yt) [reply screens/product-listing-page/product-listing-page.html:75]: png is used for transparent background.

File: screens/product-listing-page/product-listing-page.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	max-width: 30rem;
+	height: 30rem;
+	object-fit: contain;
+	background-color: var(--colour-perfect-white);
+} */
+
+.card {
+	width: auto;
```
Comment (sudipkundu999) [inline screens/product-listing-page/product-listing-page.css:41]: you may remove this, width has a default value of auto

**Review by sudipkundu999 — APPROVED** [review]
UI looks great!
Good to merge.

**Review by varun13169 — APPROVED** [review]

**Pull request merged**